tests/prepared_statment.cc

```diff
@@ -99,6 +99,11 @@ int main() {
 
 		}
 
+		auto ppsc = db << "CREATE TABLE tnums(name VARCHAR(30) PRIMARY KEY ASC NOT NULL UNIQUE, num INT NOT NULL);";
```

Comment:
This statement is never executed. Either don't save the prepared statement to execute it implicitly or call `execute`/`endr`/`operator++`.